tests/filecheck/projects/riscv-backend-paper/bottom_up.mlir

```diff
@@ -500,12 +511,16 @@ func.func public @pooling_nchw_max_d1_s2_3x3(
   }
 
 
-// CHECK:       .text
+// CHECK-NEXT:  .text
 // CHECK-NEXT:  .globl pooling_nchw_max_d1_s2_3x3
 // CHECK-NEXT:  .p2align 2
 // CHECK-NEXT:  pooling_nchw_max_d1_s2_3x3:
-// CHECK-NEXT:      mv t2, a0
-// CHECK-NEXT:      mv t0, a1
+// CHECK-NEXT:      mv t3, a0
+// CHECK-NEXT:      mv t1, a1
+// CHECK-NEXT:      li t4, -1060927488
+// CHECK-NEXT:      sw t4, -4(sp)
```

Comment:
this is the only bit that I haven't tested, but it should be ok

Reply:
Should be fine, we don't really respect the stack much anyway right now :smile: